js/src/attribute-mapping/attribute-mapping-table-categories.js

```diff
@@ -33,27 +47,19 @@ const CATEGORIES_TO_SHOW = 5;
  * @return {JSX.Element|string} The component
  */
 const AttributeMappingTableCategories = ( { categories, condition } ) => {
+	const categoryArray = categories?.split( ',' ) || [];
+	const { names } = useCategories( categoryArray );
```

Comment:
Although the final rendering result is correct, the `categoryArray` may be `['']` and `names` may be `'Category ID  (deleted)'` if a rule is like `{ "category_condition_type": "ALL", "categories": "" }`. We could consider avoiding unintended values.

Reply:
If `category_condition_type` is `ALL`, we return ` __( 'All', 'google-listings-and-ads' );`

In any other case, it will have at least one category and that issue doesn't apply as far as I understand.

Reply:
Yes, I meant we could still avoid the unintended values, they won't be used in the rendering for `ALL` rules though. Just to reduce potential problems in order to improve future maintainability.